### PR TITLE
Update deal.II version in image

### PR DESCRIPTION
**Pull request by Rombur**

### CODE REVIEW

**Comment by dalg24** [comment]
Is it just a WIP?

**Comment by Rombur** [comment]
Yeah, I want to see if the tests pass on jenkins

**Comment by masterleinad** [comment]
With the relaxed tolerances, the tests pass at least locally both with `master` and `fastap_mpi`.

File: tests/test_hierarchy_device.cu [hunk 1]
```diff
@@ -426,7 +426,7 @@ BOOST_AUTO_TEST_CASE(amgx)
     // We only supports Jacobi smoother on the device
     params->put("smoother.type", "Jacobi");
 
-    double const tolerance = 1.;
+    double const tolerance = 5.;
```
Comment (dalg24) [inline tests/test_hierarchy_device.cu]: Please call it tolerance_percent.  I jumped when I saw it initially when I thought it might be relative tolerance

Comment (dalg24) [reply tests/test_hierarchy_device.cu]: Sure

**Pull request merged**